File: Name_Generator/pig_latin.py
""" Turn a word into its Pig Latin equivalent"""
import sys


VOWELS= "aeiou"

while True:
    word = input("Type a word: ")

    if not word:
        print("No input provided. Exiting...", file=sys.stderr)
        sys.exit()
    if not word.isalpha():
        print("Please Enter a word. :)\n")
        continue

    if word[0] in VOWELS:
        pig_latin = word + 'way'
    else:
        pig_latin = word[1:] + word[0] + 'ay'
    print()
    print(f"{pig_latin}", file=sys.stderr)

    try_again = input("\nTry again? (Press Enter to close else n to stop)\n ")
    if try_again.lower == 'n':
        sys.exit()

File: Name_Generator/pig_latin.c
#include <stdio.h>
#include <string.h>
#include <ctype.h>

int main()
{
    char word[50];
    char pig_latin[54];
    const char *VOWELS = "AEIOUaeiou";

    printf("Type a word: ");
    if (scanf("%49s", word) != 1)

{
        printf("\nNo input.");
        return 1;

}

if (strchr(VOWELS, word[0]))

{
    snprintf(pig_latin, sizeof(pig_latin), "%sway", word);
}

else

{

    snprintf(pig_latin, sizeof(pig_latin), "%s%cay", word + 1, word[0]);

}

    printf("Pig Latin: %s\n", pig_latin);
    return 0;

}